pyproject.toml

```diff
@@ -39,6 +39,10 @@ invoke = "^2"
 # and for displaying advanced content such as tables, markdown, and syntax highlighted code.
 # https://rich.readthedocs.io/en/stable/introduction.html
 rich = "^13"
+# Decouple helps you to organize your settings so that you can change
+# parameters without having to redeploy your app.
+# https://github.com/HBNetwork/python-decouple
```

Comment:
Please add description about how it used for this project.
Also, maybe make it an optional dependency? Not every `saritasa-invocations` usage requires accessing of such settings